.github/workflows/ci.yml

```diff
@@ -0,0 +1,575 @@
+name: continuous-integration
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - v[0-9]+.[0-9]+.*           # i.e. v1.0, v2.1rc1
+    tags:
+      - v*
+    paths-ignore:
+      - '**/*.md'
+      - '.gitlab-ci.yml'
+      - 'CODEOWNERS'
+      - 'LICENSE'
+      - 'CODE_OF_CONDUCT.adoc'
+      - 'FILE_HEADER'
+
+env:
+  CARGO_TARGET_DIR:                /ci-cache/${{ github.repository }}/targets/${{ github.ref_name }}/${{ github.job }}
+  CARGO_INCREMENTAL:               0
+  PURELY_STD_CRATES:               ink/codegen metadata engine e2e e2e/macro ink/ir
+  ALSO_WASM_CRATES:                env storage storage/traits allocator prelude primitives ink ink/macro
+  # TODO `cargo clippy --verbose --all-targets --all-features` for this crate
+  # currently fails on `stable`, but succeeds on `nightly`. This is due to
+  # this fix not yet in stable: https://github.com/rust-lang/rust-clippy/issues/8895.
+  # Remove the following line again as soon as `clippy` on stable succeeds again.
+  CLIPPY_ALLOWED:                  clippy::extra_unused_lifetimes
+  # We plan to fully support RISC-V as a bytecode for contracts soon.
+  # RISC-V does not have a standard library in contrast to Wasm. Compiling against
+  # this target also makes sure that we don't pull in `std` by accident (through dependencies).
+  # RISC-V is a modular archtitecture. We might switch to a different flavor with more features
+  # later. For example, `riscv32imc-unknown-none-elf`.
+  RISCV_TARGET:                    riscv32i-unknown-none-elf
+
+### lint
+
+jobs:
+  spellcheck:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    container:
+      image: paritytech/ci-unified:bullseye-1.73.0
```

Comment:
```suggestion
      image: ${{ env.IMAGE }}
```

Reply:
>
I wanted to do it from the beginning, but it didn't work:
The workflow is not valid. .github/workflows/ci.yml (Line: 46, Col: 14): Unrecognized named-value: 'env'. Located at position 1 within expression: env.IMAGE